spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -266,6 +348,48 @@ partition count of the topic in the case of Kafka.
 If neither a `partitionSelectorClass` nor a `partitionSelectorExpression` is
 present the result is `key.hashCode() % partitionCount`.
 
+
+==== Overriding application properties during stream deployment
+
+Application properties that are defined during deployment override the same properties defined during the stream creation.
+
+For example, the following stream has application properties defined during stream creation:
+
+[source,bash]
+----
+dataflow:> stream create --definition "time --fixed-delay=5 | log --level=WARN" --name ticktock
+----
+
+To override these application properties, one can specify the new property values during deployment:
+
+[source,bash]
+----
+dataflow:>stream deploy ticktock --properties "app.time.fixed-delay=4,app.log.level=ERROR"
+----
+
+
+=== Deployment properties
+
+When deploying the stream, properties that control the deployment of the apps into the target platform are known as `deployment` properties.
+For instance, one can specify how many instances need to be deployed for the specific application defined in the stream using the deployment property called `count`.
+
+==== Passing instance count as deployment property
+
+If you would like to have multiple instances of an application in the stream, you
+can include a property with the deploy command:
+
+[source,bash]
+----
+dataflow:> stream deploy --name ticktock --properties "app.time.count=3"
+----
+
+Note that `count` is the *reserved* property name used by the underlying deployer. Hence, if the application also has a custom property named `count`, it is *not* supported
+ when specified in 'short-form' form during stream 'deployment' as it could conflict with the 'instance' count deployer property. Instead, the `count` as a custom application property can be
```

Comment:
nitpick but I was thinking _deployment_ (here and below), _instance_, and _creation_ (below) could be in italics instead of quotes, since they are being _emphasized_, whereas "short-form" is best handled in quotes as it is a not-so-common term that we are using (I'd say _fully-qualified_ could go either way, but to me italics seem more appropriate since it is a more common term but it's being emphasized in contrast to "short-form"... confusing? ;) )